packages/experiments/src/components/Recommendation/Recommendation.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+
+/* Dependent Components */
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import { CardFrame, ICardDropDownOption } from '../Card/CardFrame';
+
+/* Types for props and styles */
+import { IRecommendationProps, IRecommendationStyles } from './Recommendation.types';
+
+/* Styles for CardComponent and Recommendation Card */
+import { CardComponentStyles, getStyles } from './Recommendation.styles';
+
+/* Utilities */
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+
+export class Recommendation extends React.Component<IRecommendationProps, {}> {
+  /* Default Props */
+  public static defaultProps = {
+    recommendationButtonLocalizedName: 'View Recommendation',
+    recommendationButtonAriaDescription: 'Click to View Recommendation',
+    dismissRecommendationLocalizedName: 'Dismiss',
+    dismissRecommendationAriaLabel: 'Dismiss Recommendation'
+  };
```

Comment:
not used ?

Reply:
Used. Default Props are automatically substituted if the actual props are missing, they specific usage needed.

Reply:
They are used, by default. Default Props are automatically substituted if the actual props are missing, they need not be referenced specifically anywhere

---
In reply to: [196373737](https://github.com/OfficeDev/office-ui-fabric-react/pull/5252#discussion_r196373737) [](ancestors = 196373737)